Automattic/simplenote-android
# Add Email Verification

### Fix
Add email verification for users to review their account and verify their email address.  Users can also change their email address if necessary.  See the screenshots below for illustration.

![add_email_verification](https://user-images.githubusercontent.com/3827611/104940985-bbaf4580-596f-11eb-8db6-21c16e01b1a2.png)

### Test
0. Remove app from recents/overview list if present.
1. Launch app.
2. Notice ***Review Your Account*** interface is shown.
3. Tap ***X*** or back system navigation button.
4. Notice ***Review Your Account*** interface is dismissed.
5. Tap recents/overview system navigation button.
6. Tap app in recents/overview list.
7. Notice ***Review Your Account*** interface is not shown.
8. Tap recents/overview system navigation button.
9. Remove app from recents/overview list if present.
10. Launch app.
11. Notice ***Review Your Account*** interface is shown.
12. Tap ***Change Email*** button.
13. Notice web browser is shown with Simplenote login page.
14. Tap back system navigation button.
15. Tap ***Confirm*** button.
16. Notice ***Verify Your Email*** interface is shown.
17. Tap ***X*** or back system navigation button.
18. Notice ***Verify Your Email*** interface is dismissed.
19. Notice verification email is sent to email address shown.
20. Tap link in verification email.
21. Notice web browser is shown with ***Email Verified*** page.
22. Tap ***Open App*** button.
23. Notice app is shown.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated in f088432d with:
> Added reviewing, verifying, and changing account email address
